.gitignore

```diff
@@ -10,12 +10,14 @@ __pycache__/
 CMakeLists.txt
 cmake-build-debug
 venv/
+.venv/
 
 # Project-specific ignores
 build/
 expected/
 notes/
 baserom/
+baseroms/*/segments/
```

Comment:
is this rule needed? doesnt the one above cover the whole folder?

Reply:
baserom/
vs
baseroms/

That said I think this means we can remove baserom/ (moved to baseroms/VERSION/segments/)

Reply:
I keep the original `baserom` because otherwise when you pull from main git will want to add all the old files from `baserom`. This way we prevent somebody from committing those files by mistake